packages/core/src/feature-service-registry.ts

```diff
@@ -116,6 +124,22 @@ export class FeatureServiceRegistry implements FeatureServiceRegistryLike {
     private readonly options: FeatureServiceRegistryOptions = {}
   ) {}
 
+  /**
+   * Register a set of Feature Services to make them available for binding to
+   * dependent consumers.
+   *
+   * @throws Throws an error if dependencies of a registered service can't be
```

Comment:
Maybe better: "Throws an error if the dependencies of one of the given provider definitions can't be fulfilled."